input/pagecontent/3-2-1-syncerror.md

```diff
@@ -4,51 +4,38 @@ eventMaturity | [2 - Tested](3-1-2-eventmaturitymodel.html)
+2. A subscriber decides not to follow a context, returning a server conflict (409) to the Hub (see: [`Event Notification Response`](2-5-EventNotification.html#event-notification-response)) - the `severity` of the `operationoutcome` resource in the `syncerror` SHALL be `information`
+3. The Hub detects a connection issue with a subscriber (see: [`Hub Generated syncerror Events`](2-5-EventNotification.html#hub-generated-syncerror-Events)) - the `severity` of the `operationoutcome` resource shall be `information`
+
 ### Context
 
-An array containing a single FHIR OperationOutcome. The OperationOutcome SHALL use a code of `processing`. The OperationOutcome's details SHALL contain the id of the event that this error is related to as a `code` with the `system` value of `https://fhircast.hl7.org/events/syncerror/eventid` and the name of the relevant event with a `system` value of `https://fhircast.hl7.org/events/syncerror/eventname`. Other `coding` values can be included with different `system` values so as to include extra information about the `syncerror`. The OperationOutcome's `diagnostics` element should contain additional information to aid subsequent investigation or presentation to the end-user.
+The `context` array SHALL contain a single FHIR OperationOutcome.
 
 Key | Optionality | FHIR operation to generate context | Description
 ----- | -------- | ---- | ---- 
-`operationoutcome` | OPTIONAL | `OperationOutcome` | FHIR resource describing an outcome of an unsuccessful system action.
-
-The OperationOutcome SHALL use a code of `processing`.  
-The OperationOutcome's `issue[0].details.coding.code` SHALL contain the id of the event that this error is related to as a `code` with the `system` value of "https://fhircast.hl7.org/events/syncerror/eventid".  
-The OperationOutcome's `issue[0].details.coding.code` SHALL contain the name of the relevant event with a `system` value of "https://fhircast.hl7.org/events/syncerror/eventname".  
-The OperationOutcome's `issue[0].details.coding.code` SHALL contain the name of the relevant subscriber `system` value of "https://fhircast.hl7.org/events/syncerror/subscriber".  
-Other `coding` values can be included with different `system` values so as to include extra information about the `syncerror`.
-The `diagnostics` field SHALL contain a human readable explanation on the source and reason for the error.
-
-### OperationOutcome profile
-
-The profile of the OperationOutcome resource expressed in FHIR shorthand.
+`operationoutcome` | REQUIRED | `OperationOutcome` | FHIR resource describing an outcome of an unsuccessful system action.
 
-```text
+Content of the OperationOutcome resource SHALL be at least one `issue` with the initial `issue` containing:
+* `issue[0].severity` as per the above workflow section
+* `issue[0].code` of `processing`
+* `issue[0].code.details` SHALL be present
+* `issue[0].details.coding` SHALL contain at least three elements
```

Comment:
code[3] is optional, so this can not require 3.

Reply:
The third .code (code[2]) is subscriber.name and is optional; therefore, we can't "SHALL" "three elements".

Reply:
While the subscriber name is optional, this:

`issue[0].details.coding[2]` SHALL contain the optional `subscriber.name` attribute of the original subscription of the relevant subscriber with a `system` value of "https://fhircast.hl7.org/events/syncerror/subscriber"; the `code` attribute MAY be an empty string if no `subscriber.name` of the relevant subscriber was provided in the original subscription

indicates that an empty string if the subscriber.name was empty.  Thought was that Subscribers may be interested to know that the application causing the sync error has no name (and the Hub knows that it has no name so why not be specific).